### ISSUE TITLE
mutltiverseId is missing in War of the Spark

### ISSUE THREAD

**Issue by loicgasser** [mtgjson/mtgjson#321]
#### Description of Bug:

It seems that multiverseId is missing from War of the Spark. See json sample:

```json
{
"artist": "Victor Adame Minguez",
"borderColor": "black",
"colorIdentity": [
    "B",
    "R"
],
"colors": [
    "B",
    "R"
],
"convertedManaCost": 4.0,
"foreignData": [],
"frameVersion": "2015",
"hasFoil": true,
"hasNonFoil": true,
"layout": "normal",
"legalities": {
    "future": "Legal"
},
"manaCost": "{1}{B}{R}{R}",
"mcmId": 371801,
"mcmMetaId": 269255,
"name": "Widespread Brutality",
"number": "226",
"printings": [
    "WAR"
],
"purchaseUrls": {
    "cardmarket": "https://mtgjson.com/links/8e3a5569cd094ac8",
    "tcgplayer": "https://mtgjson.com/links/27c94850328dc23d"
},
"rarity": "rare",
"rulings": [],
"scryfallId": "97715d22-f432-4f67-b4ea-47b8fe6edca5",
"scryfallIllustrationId": "fa40daaf-d0fc-4f29-8246-895a58372440",
"scryfallOracleId": "3a06f4c1-2af1-495f-914f-1ac4e26f87d4",
"subtypes": [],
"supertypes": [],
"tcgplayerProductId": 187118,
"tcgplayerPurchaseUrl": "https://mtgjson.com/links/27c94850328dc23d",
"text": "Amass 2, then the Army you amassed deals damage equal to its power to each non-Army creature. (To amass 2, put two +1/+1 counters on an Army you control. If you don't control one, create a 0/0 black Zombie Army creature token first.)",
"type": "Sorcery",
"types": [
    "Sorcery"
],
"uuid": "e9359a49-b258-5544-a172-40bcfde64165",
"uuidV421": "fed14221-0e4f-57f4-a783-96bd6b18a395"
}
```

It could be a problem in scryfall as I don't know where your getting these ids from.
It is also missing there.
See:
https://api.scryfall.com/cards/search?q=Widespread%20Brutality

But the multiverseId already exists:
https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=461153

**Comment by ZeldaZach** [comment]
Will be fixed in the 4.4.0 release this weekend.

**Comment by loicgasser** [comment]
Ok thanks!
